src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -744,6 +738,151 @@ def load_ldlc_medication_effect(key: str, location: str) -> pd.DataFrame:
+    data = data[data.age_group_id.isin(valid_age_groups)]
+    data = vi_utils.normalize(data, fill_value=0)
+    data = data.filter(DEMOGRAPHIC_COLUMNS + DRAW_COLUMNS)
+    data = vi_utils.reshape(data, value_cols=DRAW_COLUMNS)
+
+    return data
+
+
+def get_re_weights_data_from_file(key: str, location: str, file_path: str) -> pd.DataFrame:
+    """Get ensemble weights for risk factors from CVD race-ethnicity project."""
+
+    key = EntityKey(key)
+    entity = get_entity(key)
+    location_id = utility_data.get_location_id(location)
+
+    data = pd.read_csv(file_path)
+    data = data.drop(["age_group_id", "sex_id", "year_id"], axis=1)
+    data = data.drop_duplicates()
+    assert len(data) == 1
+    data["rei_id"] = int(entity.gbd_id)
+    data["sex_id"] = SEX.COMBINED
+    data["age_group_id"] = 22  # all ages
+    data["measure"] = "ensemble_distribution_weight"
+
+    exposure = extract.extract_data(entity, "exposure", location_id)
+    valid_ages = vi_utils.get_exposure_and_restriction_ages(exposure, entity)
+
+    data.drop("age_group_id", axis=1, inplace=True)
+    df = []
+    for age_id in valid_ages:
+        copied = data.copy()
+        copied["age_group_id"] = age_id
+        df.append(copied)
+    data = pd.concat(df)
+
+    # define distributions not used in new R/E ensemble with weights of 0
+    non_distribution_cols = ["location_id", "rei_id", "sex_id", "measure", "age_group_id"]
+    distributions_in_data = [col for col in data.columns if col not in non_distribution_cols]
+    missing_distributions = [
+        dist for dist in DISTRIBUTION_COLUMNS if dist not in distributions_in_data
+    ]
+    for missing_distribution in missing_distributions:
+        data[missing_distribution] = 0
+
+    data = vi_utils.normalize(data, fill_value=0, cols_to_fill=DISTRIBUTION_COLUMNS)
+    data = data.filter(DEMOGRAPHIC_COLUMNS + DISTRIBUTION_COLUMNS)
+    data = vi_utils.wide_to_long(data, DISTRIBUTION_COLUMNS, var_name="parameter")
+    data = vi_utils.reshape(data, value_cols=["value"])
+
+    return data
+
+
+def transform_core_get_data_for_vivarium(
+    key: str, location: str, data: pd.DataFrame
+) -> pd.DataFrame:
+    """Take data that would be output from core.get_data and go through the processing
+    that interface.get_measure applies at this stage.
+    """
+
+    key = EntityKey(key)
+    entity = get_entity(key)
+
+    data = vi_utils.scrub_gbd_conventions(data, location)
+    validation.validate_for_simulation(data, entity, key.measure, location)
+    data = vi_utils.split_interval(data, interval_column="age", split_column_prefix="age")
+    data = vi_utils.split_interval(data, interval_column="year", split_column_prefix="year")
+    data = vi_utils.sort_hierarchical_data(data).droplevel("location")
+
+    return data
+
+
+def load_bmi_exposure(key: str, location: str) -> pd.DataFrame:
+    data = get_re_mean_exposure_data_from_me_id(key, location, data_values.BMI_MEAN_ME_ID)
```

Comment:
This function is called only once - the misdirection is confusing. I'd pull this function back